core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -462,8 +463,8 @@ public ServerBuilder tls(SslContext sslContext) {
     }
 
     /**
-     * Sets the {@link SslContext} of the default {@link VirtualHost} from the specified
-     * {@code keyCertChainFile} and cleartext {@code keyFile}.
+     * Configures SSL or TLS of  the default {@link VirtualHost} from the specified {@code keyCertChainFile}
```

Comment:
Remove extra whitespace between `of` and `the`
* `Configures SSL or TLS of  the` -> `Configures SSL or TLS of the`

Reply:
Fixed. Thanks!